kitchen/wazuh-puppet/dockerfile

```diff
@@ -1,14 +1,19 @@
 FROM vpgrp/kitchen:latest
 
 ARG source_branch=master
+ARG target_branch=master
```

Comment:
Why the same `master` value in both source and target branches?

Reply:
Hi Manu!

These are default values, and they will always be overwritten every time we build our image for testing, taking into account the corresponding task in `wazuh-jenkins`:

```yaml
  - name: DOCKER_IMAGE | BUILDING THE REQUIRED DOCKER IMAGE
    docker_image:
      name: "{{ job_name | lower }}-{{  build_num  }}"
      build:
        path: "{{ complete_path }}/kitchen/wazuh-puppet"
        args:
          source_branch: "{{ var_source_branch }}"
          target_branch: "{{ var_target_branch }}"
      source: build
    register: docler_image_output
```

Kr,

Rshad